src/packages/client/src/generation/TSClient.ts

```diff
@@ -966,6 +985,23 @@ ${aggregateTypes.length > 1
     
     `
   }
+  public toTSWithoutNamespace(): string {
+    const { model, } = this
```

Comment:
```suggestion
    const { model } = this
```
Looks like auto formatting was disabled for you?